packages/genesys-spark-components/src/components/stable/gux-form-field/functional-components/gux-form-field-label/gux-form-field-label.scss

```diff
@@ -21,6 +21,9 @@
     }
 
     &.gux-above {
```

Comment:
I think this may be causing a regression in the label for instances where there is more than one element in the label
![image](https://github.com/user-attachments/assets/9b60fa11-4c66-4dba-b828-c21d96eace5f)

Reply:
Good spot, addressed now 👍